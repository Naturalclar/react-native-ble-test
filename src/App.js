import React, { Component } from 'react';
import { BLEHelper } from './component';

class App extends Component {
  render() {
    return (
      <BLEHelper/>
    );
  }
}

export default App;