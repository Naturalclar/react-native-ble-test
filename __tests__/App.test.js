// __tests__/App.test.js
import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../src/app';

it('renders correctly', () => {
  const wrapper = shallow(<App />);
  const component = wrapper.dive();
  expect(toJson(component)).toMatchSnapshot();
});