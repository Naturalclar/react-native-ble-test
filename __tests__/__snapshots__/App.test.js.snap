// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderes correctly 1`] = `<Connect(BLEHelper) />`;

exports[`renders correctly 1`] = `<Connect(BLEHelper) />`;
